.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: rust
 dist: xenial
 
+branches:
+  only:
+    - master
```

Comment:
I'm not sure this is what we want? Won't this disable checks for PRs that aren't on a master branch?

Reply:
I just move it around, not add it.

Reply:
Well then would you be able to remove it from the changes?

Reply:
Yeah, I can. But I find putting `branches` before `script` is more accessible and readable.

Reply:
Ok, seems fine to me.